contrib/vdirsyncer/readme.md

```diff
@@ -0,0 +1,69 @@
+calcurse-vdirsyncer
```

Comment:
Can we rename this file to `README.md`? That's what we use everywhere in this project.